include/cantera/thermo/Phase.h

```diff
@@ -707,6 +707,10 @@ class Phase
     //! Return the Species object for the named species. Changes to this object
     //! do not affect the ThermoPhase object until the #modifySpecies function
     //! is called.
+    /*!
+     * @deprecated To be removed after Cantera 2.5. Replaceable with
+     * Phase::species(size_t) via Phase::speciesIndex(std::string&).
+     */
```

Comment:
I don't think this method should be deprecated.

Reply:
As far as I recall, it's no longer used within the core, but I will leave it in.